src/main/java/gov/cms/madie/models/measure/Group.java

```diff
@@ -39,5 +39,7 @@ public class Group implements GroupScoringPopulation {
   private String rateAggregation;
 
   private List<MeasureGroupTypes> measureGroupTypes;
+  
+  private Object scoringUnit;
```

Comment:
why this should be an object?

Reply:
@adongare Because UCUM uses an object with {label, value {code, name, guidance}}